types/lib/query-interface.d.ts

```diff
@@ -44,6 +44,14 @@ export interface QueryOptions extends Logging, Transactionable {
     [key: string]: string | number | string[] | number[];
   } | string[];
 
+  /**
+   * Either an object of named parameter replacements in the format `:param` or an array of unnamed
+   * replacements to replace `?` in your SQL.
+   */
+  bind?: {
```

Comment:
Can you create a type here to avoid duplication?

Reply:
Sure, done.